run/awake_start

```diff
@@ -2,13 +2,49 @@
 
 if [[ ( -z "${TESLAFI_API_TOKEN:+x}" ) && ( -z "${TESSIE_API_TOKEN:+x}" ) && ( -z "${TESLA_BLE_VIN:+x}" ) ]]
 then
-  log "Not keeping car awake using Sentry Mode."
+  log "Not configured to keep car awake."
 else
+  function nudge {
+    while true
+    do
+      sleep 300
```

Comment:
in order for the `nudge` function to work, the car will have to keep the USB port powered for an initial 5 minutes without being told to. Is that the case? I thought the USB port was powered off fairly quickly these days. Maybe it would be better to put the sleep at the end of the loop? Also, the old Tesla API "ping" would retry more quickly in case of failure, the idea being that a failure might be due to a transient network problem, and so should be retried again quickly before the car goes to sleep. Maybe worth doing for the Tessie case?